src/pages/groupComparison/GroupComparisonStore.ts

```diff
@@ -339,8 +339,8 @@ export default class GroupComparisonStore extends ComparisonStore {
                                 isSampleProfiled(
                                     sample.uniqueSampleKey,
                                     p.molecularProfileId,
-                                    this.activeMutationMapperGene!
-                                        .hugoGeneSymbol,
+                                    this.activeMutationMapperGene
+                                        ?.hugoGeneSymbol,
```

Comment:
i think you want to assert that yes there is a hugoGeneSymbol!   

What the ! means is that you the developer certify that there will be a hugoGeneSymbol.  maybe add a condition above for the negative case.